src/panels/Scene/SceneTree/SceneTreeFilters.tsx

```diff
@@ -62,15 +55,19 @@ export function SceneTreeFilters({ onFilterChange }: Props) {
             <Checkbox
               label={'Show only visible'}
               checked={showOnlyVisible}
-              onChange={(event) => setshowOnlyVisible(event.currentTarget.checked)}
+              onChange={(event) =>
+                setSpecificFilter('showOnlyVisible', event.currentTarget.checked)
+              }
```

Comment:
Hm ,not a fan of the need for a string key here. How about simply removing the `setSpecificFilter` function and just doing: 
```suggestion
              onChange={(event) =>
                setFilter({ ...filter, showOnlyVisible: event.currentTarget.checked })
              }
```